src/Database/LSMTree/Normal.hs

```diff
@@ -289,17 +290,16 @@ lookups ::
   -> TableHandle m k v blob
   -> m (V.Vector (LookupResult v (BlobRef m blob)))
 lookups ks (Internal.NormalTable th) =
-    V.mapStrict (bimap Internal.deserialiseValue BlobRef) <$!>
-    Internal.lookups const (V.map Internal.serialiseKey ks) th toNormalLookupResult
-
-toNormalLookupResult :: Maybe (Entry.Entry v b) -> LookupResult v b
-toNormalLookupResult = \case
-    Just e -> case e of
-      Entry.Insert v            -> Found v
-      Entry.InsertWithBlob v br -> FoundWithBlob v br
-      Entry.Mupdate _           -> error "toNormalLookupResult: Mupdate unexpected"
+    V.map toLookupResult <$>
```

Comment:
Is no longer using `mapStrict` intentional?

Reply:
Yes. The `V.map` could fuse in a caller.